### DISCUSSION TITLE
eBook Request App

### DISCUSSION THREAD

**Discussion by DevilsDesigns** [Archmonger/Conreq#64] [App Ideas]
I don't know if this is possible to do but I saw someone else requested it. it was said it was in the works. I don't know if you can add Mylar3, Lazy Librarian or Lidarr. But I hope it's possible to implement. Thank you ahead of time!

**Comment by Archmonger** [comment]
Renamed this to be more generic to a Conreq app for requesting ebooks.

**Comment by Archmonger** [comment]
Also to clarify, this feature is currently not being developed but there isn't any technical limitations to creating it.
